browser/net/brave_ad_block_tp_network_delegate_helper_unittest.cc

```diff
@@ -11,12 +11,58 @@
 
 #include "brave/browser/net/url_context.h"
 #include "brave/common/network_constants.h"
+#include "chrome/browser/net/secure_dns_config.h"
+#include "chrome/browser/net/stub_resolver_config_reader.h"
+#include "chrome/browser/net/system_network_context_manager.h"
+#include "chrome/common/pref_names.h"
+#include "chrome/test/base/scoped_testing_local_state.h"
+#include "chrome/test/base/testing_browser_process.h"
+#include "content/public/test/browser_task_environment.h"
+#include "content/public/test/test_browser_context.h"
 #include "net/base/net_errors.h"
 #include "testing/gtest/include/gtest/gtest.h"
 
 using brave::ResponseCallback;
 
-TEST(BraveAdBlockTPNetworkDelegateHelperTest, NoChangeURL) {
+namespace {
+class TestTorBrowserContext : public content::TestBrowserContext {
+ public:
+  TestTorBrowserContext() : content::TestBrowserContext() {}
+  ~TestTorBrowserContext() override = default;
+
+  bool IsTor() const override { return true; }
+};
+}  // namespace
+
+class BraveAdBlockTPNetworkDelegateHelperTest : public testing::Test {
```

Comment:
this fixture is not related to existing tests, so you could leave them as is and give the test class more relevant name

Reply:
fixed